Makefile

```diff
@@ -203,7 +203,7 @@ test: generate fmt vet setup-envtest $(GOTESTSUM) ## Run tests
 GINKGO_FOCUS ?= Workload cluster creation
 GINKGO_NODES ?= 3
 GINKGO_NOCOLOR ?= false
-E2E_FLAVOR ?= powervs
+E2E_FLAVOR ?= md-remediation
```

Comment:
wondering if we can make it more explicit like `powervs-md-remediation` for better understanding

Reply:
we can that will also need renaming of template file name. should we go ahead with that @mkumatag?

Reply:
I hope that's the file name we are talking about in the `tests` folder, if yes - go ahead..

Reply:
Fixed.